airflow/providers/amazon/aws/hooks/glue.py

```diff
@@ -162,8 +165,11 @@ def initialize_job(
         run_kwargs = run_kwargs or {}
 
         try:
-            job_name = self.create_or_update_glue_job()
-            return self.get_conn().start_job_run(JobName=job_name, Arguments=script_arguments, **run_kwargs)
+            if self.update_config:
+                self.create_or_update_glue_job()
+            return self.get_conn().start_job_run(
```

Comment:
Small nitpick, if you'd be so kind.  We are trying to encourage using the `self.conn` cached property over the `self.get_conn()` getter method.  If you are so inclined, could you make that change here and in the relevant test below?

Reply:
Hey @ferruzzi, sure I can change it and also other occurences in the file 👍